WeakAurasOptions/WeakAurasOptions.lua

```diff
@@ -350,7 +350,7 @@ AceGUI:RegisterLayout("ButtonsScrollLayout", function(content, children, skipLay
   end
 end)
 
-function WeakAuras.ConstructOptions(prototype, data, startorder, subPrefix, subSuffix, triggernum, triggertype, unevent)
+function WeakAuras.ConstructOptions(prototype, data, startorder, triggernum, triggertype, unevent)
```

Comment:
subPrefix and subSuffix were unused. I didn't dive into the histroy to figure when they were last used, as I can't even remember when this code last changed.